sys/include/timex.h

```diff
@@ -44,6 +44,16 @@ extern "C" {
 #define SEC_IN_USEC 1000000
 
 /**
+ * @brief The number of milliseconds per second
+ */
+#define SEC_IN_MS   (1000)
+
+/**
+ * @brief The number of microseconds per millisecond
+ */
+#define MS_IN_US    (1000)
```

Comment:
For consistency with https://github.com/RIOT-OS/RIOT/pull/2159/files#diff-83ab68e6acdc3510bfcdd1749fd2991dR44: `MS_IN_USEC`